coral-trino/src/test/java/com/linkedin/coral/trino/rel2trino/HiveToTrinoConverterTest.java

```diff
@@ -848,4 +848,16 @@ public void testSqlSelectAliasAppenderTransformer() {
         + "WHERE \"tablea\".\"a\" > 5";
     assertEquals(expandedSql, expected);
   }
+
+  @Test
+  public void testSqlSelectAliasAppenderTransformerWithoutTableAliasPrefix() {
```

Comment:
can we add a similar test in `CoralSparkTest` as well?

Reply:
Done.